custom/icds_reports/templates/icds_reports/multi_report.html

```diff
@@ -3,52 +3,91 @@
 {% load report_tags %}
 {% load i18n %}
 
-{% block reportcontent %}
-{% block pretable %}
-    {% if not report.needs_filters %}
-        {% if rendered_as == 'print' %}
-        <div class="row">
-            <div class="col-md-10">{% now "d M Y" %}</div>
-        </div>
-        <div class="row">
-            <div class="col-md-10">
-                <h3 class="media-heading">{{ title }}<br/></h3>
-            </div>
-        </div>
-        {% else %}
-        <div class="media">
-          <div class="media-body" style="height: 64px">
-            <h3 class="media-heading">{{ title }}<br/></h3>
-          </div>
-        </div>
-        {% endif %}
-    {% endif %}{{ block.super }}
+
+{% block js-inline %} {{ block.super }}
+{% if parallel_render %}
+<script>
```

Comment:
Hey, so I did try that at first, but there's been an issue. This page is return of async response, when the user clicks 'Apply' this html is inserted into the DOM by HQAsyncReport module. This page needs the URL queried. If I don't do it this way, I will need to use `initial_page_data`, but that won't work, because the intial_page_data module doesn't rescan the data elements when new elements are inserted in the way that HQAsyncReport does.

Reply:
I added a comment explaining the same